.github/workflows/unit_tests_with_featuretools_main_branch.yml

```diff
@@ -18,15 +18,15 @@ jobs:
           python-version: ${{ matrix.python_version }}
       - name: Checkout repository
         uses: actions/checkout@v2
-      - name: Build featuretools package
-        run: make package_evalml
+      - name: Update apt and install Graphviz
+        run: sudo apt update && sudo apt install -y graphviz
       - name: Set up pip
         run: |
           pip config --site set global.progress_bar off
           python -m pip install -U pip
       - name: Install EvalML & Featuretools - test requirements
         run: |
-          python -m pip install -e unpacked_sdist/[test]
+          make installdeps-test
```

Comment:
Wouldn't it be best to package EvalML and then install?

Reply:
I was worried that `make installdeps-test` wouldn't work nicely with packaged EvalML but thinking about it more it should probably work

Worth noting that this workflow seems to be the only version of the unit tests that packages EvalML

Reply:
I disagree, we package Featuretools `main` with Woodwork `main`:
https://github.com/alteryx/featuretools/blob/main/.github/workflows/unit_tests_with_woodwork_main_branch.yml#L25

Reply:
What do you disagree with?

Reply:
nvm. I get your point. Up to you if we should package